addons/knobs/src/base.js

```diff
@@ -46,7 +47,14 @@ export function object(name, value) {
 }
 
 export function select(name, options, value) {
-  return manager.knob(name, { type: 'select', options, value });
+  return deprecate(
+    manager.knob.bind(manager),
+    'in v4 keys/values of the options argument are reversed'
+  )(name, { type: 'select', options, value });
+}
```

Comment:
This should probably be rewritten as follows:

```
export const select = deprecate(
  (name, options, value) => manager.knob(name, { type: 'select', options, value }),
  'in v4 keys/values of the options argument are reversed'
);
```

See https://github.com/storybooks/storybook/pull/1745#issuecomment-362070645

Reply:
thanks! I was missing the arrow function before and passing the result of `manager.knob` 🤦